hot-reload-orchestration/src/main/kotlin/org/jetbrains/compose/reload/orchestration/OrchestrationMessage.kt

```diff
@@ -297,6 +297,39 @@ internal constructor() : OrchestrationPackage(), Serializable {
         }
     }
 
+    public data class InvalidatedComposeGroupMessage(
```

Comment:
Adding a new Serializable orchestration message will break old servers/clients. 
Let's add a Backwards & Forward compatibility test (see `ServerBackwardCompatibilityTest` & `ServerForwardCompatibilityTest`)

I expect older beta versions not to handle seeing this message gracefully. 


However, this is not a huge problem: 
I think adding a message can bump the protocol version. 
Maybe we can then add something like 'since protocol version' and just not send messages to clients with too low a protocol version 🤔 

However, if you find some other approach, feel free to propose it. 
Note: Since beta06, servers and clients will just handle such cases gracefully. The server will even distribute the message, despite not knowing its content.

Reply:
Thanks for fixing the part with checking compatibility based on version, current approach lgtm